apps/web/src/pages/dashboard/index.tsx

```
@@ -1,12 +1,16 @@
+import ErrorBoundary from "@src/components/commons/ErrorBoundary";
+import Skeleton from "@src/components/commons/Skeleton";
+import Error from "@src/pages/seats/Error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)